docs/modules/ROOT/pages/pallets/balances.adoc

```diff
@@ -0,0 +1,233 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= pallet_balances
+
+Branch/Release: `release-polkadot-v1.3.0`
+
+== Purpose
+
+The Balances pallet provides functions for:
+
+* Getting and setting free balances.
+* Retrieving total, reserved, and unreserved balances.
+* Repatriating a reserved balance to a beneficiary account that exists.
+* Transferring a balance between accounts (when not reserved).
+* Slashing an account balance.
+* Account creation and removal.
+* Managing total issuance.
+* Setting and managing locks.
```

Comment:
`ExistentialDeposit` is explained in `Config` part. But if you feel like it should be mentioned here as well (I think it would be duplication), I can put another sentence. 
Let me know :)